app/frontend/src/components/Answer/SpeechOutputBrowser.tsx

```diff
@@ -15,19 +17,25 @@ try {
     console.error("SpeechSynthesis is not supported");
 }
 
-const getUtterance = function (text: string) {
+const getUtterance = function (text: string, lngCode: string) {
     if (synth) {
         const utterance = new SpeechSynthesisUtterance(text);
-        utterance.lang = "en-US";
+        utterance.lang = lngCode;
         utterance.volume = 1;
         utterance.rate = 1;
         utterance.pitch = 1;
-        utterance.voice = synth.getVoices().filter((voice: SpeechSynthesisVoice) => voice.lang === "en-US")[0];
+        utterance.voice = synth.getVoices().filter((voice: SpeechSynthesisVoice) => voice.lang === lngCode)[0];
```

Comment:
Will this always find a matching voice? I'm worried it won't, and then there'll be an error. Could it default to en-US in that case?

Reply:
Thank you for the feedback. I have addressed the concern regarding defaulting to "en-US" if a matching voice for lngCode is not found.